src/components/Forms/BaseInput.jsx

```diff
@@ -11,9 +11,10 @@ const sizes = {
 };
 
 export const BaseInput = ({ as: Tag, size = "medium", isError, className, isRequired, value, ...rest }) => {
+    const stringValue = isString(value) && value.replace("$", "").trim();
```

Comment:
Not all currencies are `$` See all these variations. New values not in this list may also come. You need to find a more generic solution https://ui.xola.io/?path=/story/data-display-currency--all-currencies

Also check what component is using `Input` as `$1.23` (string). Maybe it can use `apps/seller/src/components/Input/CurrencyInput.tsx:6` instead

Reply:
changed property in the replace method to return numbers only